test/plots/wheelPlotTests.ts

```diff
@@ -0,0 +1,304 @@
+        let radiusLimit = Math.min(SVG_WIDTH, SVG_HEIGHT) / 2;
+        assert.deepEqual(rScale.range(), [0, radiusLimit] , "the radius scale range is set properly");
+
+        svg.remove();
+      });
+
+      it("undefined, NaN and non-numeric strings not be represented in a Pie Chart", () => {
+        let data = [
+          { r1: 0, r2: 1, t1: 0, t2: 180 },
+          { r1: undefined, r2: 2, t1: 180, t2: 360 },
+          { r1: NaN, r2: 2, t1: 180, t2: 360 },
+          { r1: "Bad String", r2: 2, t1: 180, t2: 360 },
+          { r1: 1, r2: undefined, t1: 180, t2: 360 },
+          { r1: 1, r2: NaN, t1: 180, t2: 360 },
+          { r1: 1, r2: "Bad String", t1: 180, t2: 360 },
+          { r1: 1, r2: 2, t1: 180, t2: 360 },
+          { r1: 1, r2: 2, t1: undefined, t2: 360 },
+          { r1: 1, r2: 2, t1: NaN, t2: 360 },
+          { r1: 1, r2: 2, t1: "Bad String", t2: 360 },
+          { r1: 1, r2: 2, t1: 180, t2: undefined },
+          { r1: 1, r2: 2, t1: 180, t2: NaN },
+          { r1: 1, r2: 2, t1: 180, t2: "Bad String" },
+          { r1: 2, r2: 3, t1: 90, t2: 180 }
+        ];
+
+        let dataset = new Plottable.Dataset(data);
+        wheelPlot.addDataset(dataset);
+        wheelPlot.renderTo(svg);
+
+        let elementsDrawn = (<any> wheelPlot)._element.selectAll(".arc");
+
+        assert.strictEqual(elementsDrawn.size(), 3,
+          "There should be exactly 3 sectors in the wheel chart, representing the valid values");
+
+        assert.lengthOf(wheelPlot.entities(), 3,
+          "There should be exactly 3 entities in the wheel chart, representing the valid values");
+
+        svg.remove();
+      });
+    });
+
+    describe("innerRadius and outerRadius", () => {
+      let SVG_WIDTH = 400;
+      let SVG_HEIGHT = 500;
+      let svg: d3.Selection<void>;
+      let wheelPlot: Plottable.Plots.Wheel<number, number>;
+      let rScale: Plottable.Scales.Linear;
+      let tScale: Plottable.Scales.Linear;
+      let data: [any];
+      let dataset: Plottable.Dataset;
+
+      beforeEach(() => {
+        svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
+        rScale = new Plottable.Scales.Linear();
+        tScale = new Plottable.Scales.Linear();
+        tScale.domain([0, 360]);
+        wheelPlot = new Plottable.Plots.Wheel();
+        wheelPlot.startAngle((d) => d.t1, tScale);
+        wheelPlot.endAngle((d) => d.t2, tScale);
+        data = [
+          {r1: 0, r2: 1, t1: 0, t2: 180 },
+          {r1: 1, r2: 2, t1: 180, t2: 360 }];
+        dataset = new Plottable.Dataset(data);
+        wheelPlot.addDataset(dataset);
+      });
+
+      it("can set and get innerRadius", () => {
+        wheelPlot.outerRadius((d) => d.r2, rScale);
+        assert.isUndefined(wheelPlot.innerRadius(), "innerRadius is initiized to undefined");
+
+        wheelPlot.innerRadius(0);
+        wheelPlot.renderTo(svg);
+        let constInnerRadiusAccessor = (<any> Plottable.Plot)._scaledAccessor(wheelPlot.innerRadius());
```

Comment:
Rather than `<any>`-casting in to the `Plot`, shouldn't you call `innerRadius()` as a getter and check that it returns the correct value? In this case, it should return an `AccessorScaleBinding` with no `Scale` and an `Accessor` that returns a constant value.